core/sis-utility/src/main/java/org/apache/sis/util/logging/LoggerAdapter.java

```diff
@@ -571,6 +571,7 @@ public void logp(final Level level, final String sourceClass, final String sourc
      *
      * @since 0.5
      */
+    @Override
```

Comment:
In those particular cases, this annotation can not be added on a code targeting JDK7 platform, because this method had been added in JDK8.

Reply:
Undoing.